src/common/out.h

```diff
@@ -53,9 +53,15 @@
 #define OUT_FATAL_DISCARD_NORETURN
 #endif
 
+#if defined(DEBUG) || defined(__clang_analyzer__) || defined(__COVERITY__)
```

Comment:
Please wrap this with #ifndef EVALUATE_DBG_EXPRESSIONS.

Reply:
done

Reply:
No, that's not what I meant. You are overwriting EVALUATE_DBG_EXPRESSIONS.

Reply:
done :P